test/pytest/conftest.py

```diff
@@ -12,6 +12,7 @@
 # Exclude the following tests from regression tests
 collect_ignore = []
 collect_ignore.append("test_example_torchrec_dlrm.py")
+collect_ignore.append("test_example_streaming_video.py")
```

Comment:
should we ignore this? How long does the test take to run?

Reply:
The current tests I have added are not adding that much value to the regression test. So, not sure if its worth it. 
We might need a test to actually verify that APIs of requests-futures or OpenCV being used are not failing. But for that, we would need to install these libraries which makes the size of ts installation bigger. We need to think of a better solution to handle the tests for examples.